contents/naturalLanguage/nl.md

```diff
@@ -0,0 +1,198 @@
+
+### Applications of NLP
+
+#### Input methods editors, e.g. Chinese, Japanese and Korean
+Modern input method editors(IMEs) do more than simply translating input to output in the respective language.
+By tackling the syntactic and relational elements of natural language, IMEs provide autocomplete functionality by suggesting words that are related or occur together frequently. 
+
+For example, typing "*beijing*" in Moon IME, a Chinese IME, will bring up "*olympics*" as a possible suggestion<sup>[12](#footnote12)</sup>.
+
+#### Indexing and information retrieval, e.g. Google's page rank and normalization of search terms
+Search terms on Google are grouped together by semantic similarity in real time at volume to identify trends worldwide. 
+This is achieved with techniques similar to **word2vec**. 
+
+#### Aggregation and clustering of documents, e.g. Cambridge Analytica, Google News
+Techniques like **doc2vec**<sup>[13](#footnote13)</sup> build upon **word2vec** and provide a representation of paragraphs in vector space.
+Similar to **word2vec**, this enables classification of documents and paves the road to powerful information retrieval techniques. 
+
+#### Machine translation, e.g. Google Translate
+**seq2seq**<sup>[3](#footnote3)</sup>, a algorithm developed by Google, transforms a sequence to another. 
+This family of techniques uses special <tooltip content="artifical neural networks a computer system made up of a number of simple, highly connected processing elements which translate information to some form of desired output"><i>artificial neural network</i></tooltip>   architectures to model sentences.  
+
+#### Automated customer support, e.g. ChatBot, NanoRep
+Human-curated databases (e.g. **WordNet**) are used together with techniques like **word2vec** to extract actionable words or phrases<sup>[14](#footnote14)</sup>. 
+For example, the two possible input from users regarding a replacement for a digital banking token: 
+
+> * I want to replace my token.
+> * My old token is broken. 
+
+can be mapped back to the same intent to which a predefined response can be given. 
+
+<p align="center">
+    <pic src="dbsChatbot.png" alt="DBS chatbot responding to two different messages with a similar intent" width="90%">
+      <sub><i>Figure 6. DBS chatbot responding to two different messages with a similar intent</i></sub>
+    </pic>
+</p>
+
+## What's Next
+1. **Get an intuition for linguistics**
+
+    [The Language Instinct](https://stevenpinker.com/publications/language-instinct) by Steven Pinker provides accessible insight about how humans learn language and the basics of formal linguistics. 
+    Learn about how to deconstruct language into formal structures for critical analysis through engaging examples.
+    
+1. **Visually explore the themes present in NLP**
+    
+    Stanford's [CoreNLP](https://corenlp.run/) provides a visual representation of NLP techniques.
```

Comment:
I couldn't get this to run though

Reply:
It can be a bit sluggish to load at times depending on the load of the server.